tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -193,9 +193,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                     element: link,
                     jQueryTrigger: "focus"
                 });
-
-                
-                assertVisibleTips("There should be tooltip visible when none of the pageLinks are focused", [linkId]);
+                assertVisibleTips("The tooltip for page link " + (idx + 1) + ", in the " + location + " page bar is visible", [linkId]);
```

Comment:
Could you move the `idx + 1` to an appropriately named variable. Looking at this line is a bit hard to understand because it's not immediately apparent what the `idx +1` is for and the overloaded use of `+` for addition and string concatenation.